pfcpiface/up4_test.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package pfcpiface
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_UP4_initTunnelPeerIDs(t *testing.T) {
+	type args struct {
+		mockUP4 *UP4
+	}
+
+	type want struct {
+		length int
+	}
+
+	tests := []struct {
+		name string
+		args *args
+		want *want
+	}{
+		{
+			name: "init tunnelPeerIDPool",
+			args: &args{
+				mockUP4: &UP4{},
+			},
+			want: &want{
+				length: maxGTPTunnelPeerIDs,
+			},
```

Comment:
I'd skip the table-based approach if there can only one sensible case.

Reply:
It depends on the function under test. Good indicators against test tables are

- only one table entry
- no input arguments
- no or simple boolean return/output parameters